### DISCUSSION TITLE
Different Types of Map Overlays

### DISCUSSION THREAD

**Discussion by mr1716** [mattieserver/netbox-topology-views#490] [Ideas]
Hey, wanted to see if there were ways to overlay different maps to the back. Like if I have sites and regions in different cities, it would be interesting and very useful to have the sites shown on a map of the world, etc...

**Comment by mattieserver** [comment]
This has been asked a lot but we won't do it because the current implementation would not work when changing the zoom level.

**Reply by mr1716** [reply]
Thanks. Is it something on the roadmap to implement in a future version of the plug-in @mattieserver ?

**Reply by mattieserver** [reply]
No. This is not planned.
